### PR TITLE
Add hotspot rule S2255 (Cookies)

### LINKED ISSUE #1139: Rule S2255: Using cookies is security-sensitive
Implements [RSPEC-2255](https://jira.sonarsource.com/browse/RSPEC-5084)

### PR DESCRIPTION
**Pull request by vilchik-elena**
Fixes #1139

### CODE REVIEW

File: eslint-bridge/src/rules/cookies.ts [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    let usingExpressFramework = false;
+
+    return {
+      Program() {
+        // init flag for each file
+        usingExpressFramework = false;
+      },
+
+      Literal(node: estree.Node) {
+        if ((node as estree.Literal).value === "express") {
```
Comment (andrea-guarino-sonarsource) [inline eslint-bridge/src/rules/cookies.ts]: IMO, we need to be more precise: I think we should check for `import` or `require` statements that contains "express"

File: eslint-bridge/src/rules/cookies.ts [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+            message,
+            node,
+          });
+        }
+      },
+
+      CallExpression(node: estree.Node) {
+        const { callee, arguments: args } = node as estree.CallExpression;
+        if (
+          callee.type === "MemberExpression" &&
```
Comment (andrea-guarino-sonarsource) [inline eslint-bridge/src/rules/cookies.ts]: what about having a `usingHttp` flag as well?

File: eslint-bridge/src/rules/cookies.ts (hunk 1 above)
Comment (vilchik-elena) [reply eslint-bridge/src/rules/cookies.ts]: Doing that will overcomplexify implementation without a lot of return value. I believe we will not have FP even like this.

File: eslint-bridge/src/rules/cookies.ts (hunk 2 above)
Comment (vilchik-elena) [reply eslint-bridge/src/rules/cookies.ts]: We are checking first argument value, so pattern is pretty narrow, I don't think we need more specialization

**Pull request merged**